tests/ui/attributes/empty-repr.stderr

```diff
@@ -4,6 +4,7 @@ error: unused attribute
 LL | #[repr()]
    | ^^^^^^^^^ help: remove this attribute
    |
+   = note: attribute `repr` with an empty list has no effect
```

Comment:
This feels a bit weird. "has no effect" is a non-sequitur, because it's a compile error.

Reply:
i.e. only for the test its an error

Reply:
this isn't an error, it's a warning. It's simply emitted as an error because of `#[deny(unused_attributes)]`